setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+meta = {}
+exec(open('meta.py').read(), meta)
+
+setup(
+    name=meta.get('NAME'),
+    version=meta.get('VERSION'),
+    description=meta.get('DESCRIPTION'),
+    long_description=''.join(open('docs/Virt-v2v-wrapper.md').readlines()),
+    keywords=meta.get('KEYWORDS'),
+    author=meta.get('AUTHOR'),
+    author_email=meta.get('EMAIL'),
+    license=meta.get('LICENSE'),
+    packages=find_packages(),
+    entry_points={
+        'console_scripts': [
+            'virt-v2v-wrapper.py = wrapper.virt_v2v_wrapper:main'
+        ]
+    },
+    install_requires=[
+        'pycurl'
+    ],
+    extras_require=[
+        'ovirt-engine-sdk-python',
+    ],
```

Comment:
Travis fails on this with:
```
   error in virt-v2v-wrapper setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
```

Reply:
OH, that's right, I just needed to move it out from the default requirements so that it is not installed when building the kubevirt container so that it does not drag in unnecessary things.  Then did not really try it out =)  Will fix that, thanks.